common/src/main/java/net/william278/huskclaims/claim/ClaimWorldSerializer.java

```diff
@@ -109,7 +109,8 @@ protected static Claim getUpgradableClaim(@NotNull JsonObject co, @NotNull Gson
         childrenArray.forEach(c -> children.add(getUpgradableClaim(c.getAsJsonObject(), gson)));
         final Set<OperationType> defaultFlags = gson.fromJson(co.getAsJsonArray("default_flags"), GsonProvider.OPERATION_TYPE_SET_TOKEN);
         final boolean inheritParent = co.get("inherit_parent").getAsBoolean();
-        return new Claim(owner, region, trustedUsers, trustedGroups, trustedTags, Maps.newConcurrentMap(), children, inheritParent, defaultFlags);
+        final boolean privateClaim = co.get("private_claim").getAsBoolean(); 
```

Comment:
This logic here is for upgrading claims from older data versions - hence this isn't needed as this is a new feature; let's get rid of this and just use "false" for the constructor call :)

Reply:
👍